static/index.html

```diff
@@ -37,6 +38,7 @@
     <!-- Adapter -->
     <script type="text/javascript" src="js/adapter.js"></script>
     <link rel="stylesheet" type="text/css" href="css/adapter.css" />
+    <script type="text/javascript" src="js/check-login"></script>
```

Comment:
This is missing the `.js`. I also don't think that this page needs the ability to login